### DISCUSSION TITLE
Issues, branches and pull requests

### DISCUSSION THREAD

**Discussion by w2vy** [rchain-community/rgov#221] [General] [answered]
I have made some changes in Issue #215 that are needed to run any test scripts.
So in order to work on anything else I need 215 closed out, since to date I have always only made branches off of master.

Can I make a new branch off of branch pyrgov_updated_checkBalance.rho to make additional testing changes?

Are there any preferences to how many problems or improvements should be included in a single issue?
Can/should a branch contain updates for more than one issue?
Can/should a pull request include more than one issue?

@jimscarver @Bill-Kunj @dckc

**Comment by dckc** [comment]
> I have made some changes in Issue #215 that are needed to run any test scripts.

#215 is an issue; aren't changes made in PRs? I don't understand this much.

Also, I don't understand #215. I'll ask about it there.

**Reply by w2vy** [reply]
If you look at #215 there is a pull request #216, which are being used as an illustration of the situation I am asking about

The subject is "Issues, branched and pull requests"

So far I have only created a branch off of master to perform work.
The primary question is that if I fix a problem, for example the Issue #215 and PR #216, and any additional work I do totally depends on PR #216 being approved then is it acceptable to create a new branch of the branch THAT work was done on?

My only other choice would be to do nothing at all while waiting for the PR to be approved

**Reply by dckc** [reply]
yes, branching from some branch other than master is sometimes The Right Thing to do. This sounds like such a time.

**Reply by Bill-Kunj** [reply]
Time to close this?

**Comment by w2vy** [comment]
Yes

On July 10, 2021 6:58:20 PM EDT, Bill-Kunj ***@***.***> wrote:
>Time to close this?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/rchain-community/rgov/discussions/221#discussioncomment-989271

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by Bill-Kunj** [comment, marked as answer]
This is answered by https://github.com/rchain-community/rgov/discussions/221#discussioncomment-989306
